lib/models/auth/vm_email_auth_options_page.dart

```diff
@@ -0,0 +1,61 @@
+library vm_email_auth_options_page;
```

Comment:
this was renaming vm_other_auth_options_page.dart to vm_email_auth_options_page.dart.
vm_other_auth_options_page.dart was deleted and we added the autovalidate member :)

Reply:
thanks for the helpful commenting!